haskell-tools.nvim-scm-1.rockspec

```diff
@@ -0,0 +1,40 @@
+local MODREV, SPECREV = 'scm', '-1'
+rockspec_format = '3.0'
+package = 'haskell-tools.nvim'
+version = MODREV .. SPECREV
+
+description = {
+  summary = 'Supercharge your Haskell experience in neovim!',
+  detailed = [[
+    Automatically configures the haskell-language-server neovim client
+    and integrates with other haskell tools, such as hoogle and fast-tags.
+  ]],
+  labels = { 'neovim', 'lsp', 'haskell', 'plugin' },
+  homepage = 'https://github.com/MrcJkb/haskell-tools.nvim',
+  license = 'GPL-2.0',
+}
+
+dependencies = {
+  'lua >= 5.1',
+  'nvim-lspconfig',
+  'plenary.nvim',
+  'telescope.nvim',
```

Comment:
:thinking:  `telescope.nvim`  requires a `require('telescope').setup()` call.
Since it's an optional dependency of this plugin, i"m not sure if I should add it as a rockspec dependency.

Reply:
Yup, me too.